src/Data/ImportPath.hs

```diff
@@ -19,7 +19,7 @@ data ImportPath = ImportPath { unPath :: FilePath, pathIsRelative :: IsRelative
 importPath :: Text -> ImportPath
 importPath str = let path = stripQuotes str in ImportPath (T.unpack path) (pathType path)
   where
-    pathType xs | not (T.null xs), T.head xs == '.' = Relative -- head call here is safe
+    pathType xs | not (T.null xs), fmap fst (T.uncons xs) == Just '.' = Relative -- head call here is safe
```

Comment:
I don’t think this is an improvement. That said, if we decide to keep it, we should 🔥 the `T.null` guard immediately before it.

Reply:
I’ll refactor this a bit.

Reply:
This is a bit more readable now, eliding the `null` and `head` in favor of an `uncons` brought out into its own def.